ntc_templates/templates/cisco_ios_show_tacacs.textfsm

```diff
@@ -24,6 +24,9 @@ Start
   ^\s+Failed\s+Connect\s+Attempts:\s+${FAILED_CONNECTIONS}\s*$$
   ^\s+Total\s+Packets\s+Sent:\s+${PACKETS_SENT}\s*$$
   ^\s+Total\s+Packets\s+Recv:\s+${PACKET_RECEIVED}\s*$$
+  ^\s+Server\s+Status:\s+\S+\s*$$
+  ^Continous\s+Authc\s+fail\s+count:\s+\d+\s*$$
```

Comment:
```suggestion
  ^Continous\s+Auth\w+\s+fail\s+count:\s+\d+\s*$$
```

Combine the two Continuous Auth lines into a single one by using regexes.

Reply:
I'm also going to suggest that we put optional white space at the beginning of the Continuous lines just in case (there's ever any longer lines) causing the output to change.